longevity_test.py

```diff
@@ -77,11 +92,11 @@ def test_custom_time(self):
         self.run_pre_create_keyspace()
         self.run_pre_create_schema()
 
-        if fullscan_params := self._get_scan_operation_params(scan_operation='run_fullscan'):
-            self.run_fullscan_thread(ks_cf=fullscan_params['ks_cf'], interval=fullscan_params['interval'])
-
-        if full_partition_scan_params := self._get_scan_operation_params(scan_operation='run_full_partition_scan'):
-            self.run_full_partition_scan_thread(**full_partition_scan_params)
+        if scan_operation_params := self._get_scan_operation_params():
```

Comment:
this if seems redundant. In case of empty scan operation params list, for loop skips anyway